hack/common.sh

```diff
@@ -48,11 +48,7 @@ LATEST_ROOK_IMAGE="rook/ceph:v1.9.0"
 LATEST_NOOBAA_IMAGE="noobaa/noobaa-operator:master-20210912"
 LATEST_NOOBAA_CORE_IMAGE="noobaa/noobaa-core:master-20210912"
 LATEST_NOOBAA_DB_IMAGE="centos/postgresql-12-centos7"
-# The stretch cluster feature will come in ceph pacific(v16).  We don't have an
-# image for it yet. Meanwhile, we will use an image that has the required
-# patches. This is required for the CI and does not impact anything else.
-# TODO: revert to using ceph/ceph image once v16 is out.)
-LATEST_CEPH_IMAGE="ceph/daemon-base:latest-pacific"
+LATEST_CEPH_IMAGE="quay.io/ceph/ceph:v16"
```

Comment:
```suggestion
LATEST_CEPH_IMAGE="ceph/ceph:v16"
```
default registry is quay.io.